Proto/cg.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
```

Comment:
@matthid Possibly, or some core library that's inside FAKE.  

I suspect you should become a co-maintainer on FAKE if you want to push it that way?  It's in need of a second maintainer (I think @forki appreciates that).